.tractusx

```diff
@@ -0,0 +1,3 @@
+product: "SSI Dim Wallet Stub"
+leadingRepository: "https://github.com/eclipse-tractusx/ssi-dim-wallet-stub"
+repositories: []
```

Comment:
the file is incomplete see https://github.com/eclipse-tractusx/tractus-x-umbrella/blob/main/.tractusx

Reply:
Addressed with: https://github.com/eclipse-tractusx/ssi-dim-wallet-stub/pull/1/commits/6a3e46e6ebbc0ed1cdc64e8ad1c3b1c15c388d11